storage/Devicegraph.cc

```diff
@@ -43,13 +44,15 @@ namespace storage
 
 
     WrongNumberOfParents::WrongNumberOfParents(size_t seen, size_t expected)
-	: Exception(sformat("wrong number of parents, seen '%zu', expected '%zu'", seen, expected))
+	: Exception(sformat("wrong number of parents, seen '%zu', expected '%zu'", seen, expected),
+		    Silencer::is_any_active() ? DEBUG : WARNING)
```

Comment:
I just would like to know contract. E.g. if it always change to DEBUG then it can have method like Silence.silence(WARN) which return WARN or DEBUG depending if enabled.